README.md

```diff
@@ -182,7 +185,8 @@ the VMware Aria Operations Trust Store, the user will be prompted to either acce
 the certificate. If it is accepted, the certificate will be added to the AdapterInstance
 object that is passed to the 'test' and 'collect' methods. Any certificate that is
 encountered in those methods should then be validated against the certificate(s)
-in the AdapterInstance.
+in the AdapterInstance. This method will not only work against HTTPS endpoints, different types
+of endpoint will not work and there is no benefit of returning them.
```

Comment:
We should give examples of other types of endpoints

'no benefit of returning them' this seems redundant as we just said they won't work.